### PR TITLE
[Snyk + GH Actions] Update requirements

### PR DESCRIPTION
**Pull request by nickumia-reisys**
Update requirements
- Updated `requirements.in` + `requirements.txt`
- Auto-generated by [snyk.yml][1]

[1]: https://github.com/gsa/inventory-app/.github/workflows/snyk.yml

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
-boto3==1.26.73
-botocore==1.29.73
+boto3==1.26.115
+botocore==1.29.115
 certifi==2022.12.7
 cffi==1.15.1
 chardet==3.0.4
 ckan==2.9.8
-ckanext-datajson==0.1.15
+ckanext-datajson==0.1.18
```
Comment (jbrown-xentity) [inline requirements.txt:13]: This is weird; normally we bump this ourselves. Why have we gotten so out of date?

File: requirements.txt [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
 ckanext-dcat-usmetadata==0.3.9
-ckanext-envvars==0.0.2
+ckanext-envvars==0.0.3
```
Comment (jbrown-xentity) [inline requirements.txt:15]: Wow, I didn't realize they updated this last year... https://pypi.org/project/ckanext-envvars/#history I thought it hadn't changed in 8 years.

File: requirements.txt [hunk 3]
```diff
@@ -1,29 +1,29 @@
...
-ckanext-s3filestore @ git+https://github.com/keitaroinc/ckanext-s3filestore.git@8e815a47cd902884ff93fcff96d90349fb6c5459
+ckanext-s3filestore @ git+https://github.com/keitaroinc/ckanext-s3filestore.git@caf88c0352ffe7b4432d3d55ddfb0a71249ceddd
 ckanext-saml2auth @ git+https://github.com/keitaroinc/ckanext-saml2auth.git@b2c6cfc2f9d6179601017f2d6530cf4ee5331f00
 ckanext-usmetadata==0.2.6
--e git+https://github.com/ckan/ckanext-xloader.git@9db532439dbb8b88f7c9d11f1e573ff6c164e3b9#egg=ckanext_xloader
+-e git+https://github.com/ckan/ckanext-xloader.git@c062f547e8e5dd80fd5dae6d184fe268ccb4b32e#egg=ckanext_xloader
 ckantoolkit==0.0.7
 click==7.1.2
-cryptography==39.0.1
+cryptography==40.0.2
```
Comment (jbrown-xentity) [inline requirements.txt:23]: Gives me a little pause, but tests pass...

**Review by jbrown-xentity — COMMENTED** [review]
Approving, but leaving for @robert-bryson to merge and watch in case something goes wrong; this is a lot of changes and we should have O&M monitor.

File: requirements.txt (hunk 1 above)
Comment (nickumia-reisys) [reply requirements.txt:13]: We had harvesting updates that didn't effect inventory and that's why we never looked to update it here.

File: requirements.txt (hunk 2 above)
Comment (nickumia-reisys) [reply requirements.txt:15]: It was because of CKAN 2.10 breaking changes.. so yeah haha..

**Comment by nickumia-reisys** [comment]
This was a good PR.. but it's since been overwritten and can no longer be used..

**Pull request closed without merging**